src/pages/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as Login } from "./Login";
+export { default as Register } from "./Register";
```

Comment:
We should use the following syntax
```import Login from './Login'
import Register from './Register'
export default {
Login,
Register
}```

Reply:
Done with a little edition: 
```ts
export { Login } from "./Login";
export { Register } from "./Register";
```
Because my exports are now named exports, allowing me to import those components with the following synthax:
```ts
import { Login, Register } from "pages";
```